### PR TITLE
Do not produce a new payload if low stat

### PR DESCRIPTION
**Pull request by argiro**
With the current behaviour, the Ecal pedestal PCL generates a new payload even if the stat is insufficient. The new payload is identical to the old.
With the new behaviour, a new payload is _not_ created if statistics is low.

### CODE REVIEW

**Comment by lpernie** [comment]
please test

File: Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc [hunk 1]
```diff
@@ -106,8 +116,19 @@ void ECALpedestalPCLHarvester::dqmEndJob(DQMStore::IBooker& ibooker_, DQMStore::
     }
 
 
+    bool enough_stat = false;
+    if ((kBarrelSize-skipped_channels_EB)/kBarrelSize > threshChannelsAnalyzed_ &&
```
Comment (lpernie) [inline Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc]: You do not want to do an OR here?

**Comment by argiro** [comment]
bah, we really wanted enough stat bot in barrel and in endcap. It's actually irrelevant, as EB and EE take data at the same time ...

**Comment by lpernie** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**